docs/_component/editor.client.js

```diff
@@ -53,6 +57,12 @@ function useMdx(defaults) {
             recmaPlugins: [capture('esast')]
           })
         ).default
+
+        if (!config.position) {
+          removePosition(file.data.mdast, {force: true})
+          removePosition(file.data.hast, {force: true})
+          removePosition(file.data.esast, {force: true})
```

Comment:
BTW not sure this works for esast, is there a tool to remove esast positions to make exploring the AST more convenient?

I mean these nodes:

```js
"start": 68,
                                            "end": 75,
                                            "loc": {
                                              "start": {
                                                "line": 4,
                                                "column": 13,
                                                "offset": 68
                                              },
                                              "end": {
                                                "line": 4,
                                                "column": 20,
                                                "offset": 75
                                              }
                                            },
```

Reply:
You can try [`unist-util-position-from-estree`](https://github.com/syntax-tree/unist-util-position-from-estree).

Reply:
@remcohaszing that only gets a position, it doesn’t clear start/end/loc

Reply:
Replaced with:

```js
import {visit as visitEstree} from 'estree-util-visit'

export function removePositionEsast(tree) {
  visitEstree(tree, remove)
  return tree

  function remove(node) {
    delete node.loc
    delete node.start
    delete node.end
    delete node.range
  }
}
```